tests/test-sources/plugins/bufferlines/bufferline.nix

```diff
@@ -124,10 +125,85 @@
     };
   };
 
-  no-packages = {
-    plugins.bufferline = {
-      enable = true;
-      iconsPackage = null;
+  default-packages =
+    { config, ... }:
+    {
+      plugins.bufferline = {
+        enable = true;
+      };
+
+      assertions = [
+        {
+          assertion =
+            config.extraPlugins != [ ]
+            -> lib.any (
```

Comment:
I think this should be `&&`. Using `->` here will be true when `extraPlugins` is empty, and will only check the `lib.any` when it is not empty.

You can read `->` as "implies". A logical implication is true when the left-hand-side is false, but when the lhs is true, it also checks the right-hand-side.

`a -> b` is equivalent to `!a || b`.

Reply:
```suggestion
            config.extraPlugins != [ ]
            && lib.any (
```